crates/wallet/src/test_utils.rs

```diff
@@ -160,6 +160,13 @@ pub fn get_test_single_sig_cltv() -> &'static str {
     "wsh(and_v(v:pk(cVpPVruEDdmutPzisEsYvtST1usBR3ntr8pXSyt6D2YYqXRyPcFW),after(100000)))"
 }
 
+/// `wsh` descriptor with policy `and(pk(A),after(500_000_000))`
+/// the parameter passed to miniscript fragment `after` has to equal or grather than 500_000_000
```

Comment:
```suggestion
/// the parameter passed to miniscript fragment `after` has to equal or greater than 500_000_000
```

Reply:
@f3r10 I don't think the extra commentary is needed, but we can make it a code comment as a note to developers and leave the doc string as a single line:

```rs
/// `wsh` descriptor with policy `and(pk(A),after(1734230218))`
```

Can you give this helper a better name, like `get_test_single_sig_cltv_timestamp`. I agree with the suggestion to use a realistic timestamp (without underscores).

Reply:
@ValuedMammal fixed with your comments.

However, based on what we talked about at the previous meeting, I was about to close this PR. 
Do you think that this test is testing what is supposed to?

Maybe when the implementation of the planning module that you are working on is finished, this test will be more relevant. :thinking:

Reply:
yes I think we should have the test, I'm not sure of another way to test that time based timelocks are satisfied.